commands/silent-stderr

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env bash
 
+# fish also supports: "$@" ^ /dev/null
+
+# help
+if is-help "$@"; then
+	cat <<-EOF >/dev/stderr
+		ABOUT:
+		    Send output to dev null.
```

Comment:
`Send stderr to /dev/null`
or
`Silence stderr output, but writing it to /dev/null`